### PR TITLE
Add MagicLink support through passwordless module

### PR DESCRIPTION
**Pull request by zlwaterfield**
Add Passwordless support for MagicLink. Adds two methods:
- `create_session` - generate a passwordless session for a user from their email
- `send_session` - send an email to a user with a Magic Link confirmation URL

### CODE REVIEW

**Comment by zlwaterfield** [comment]
Tested locally using a sandbox API key and it is working.

File: workos/passwordless.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+            "passwordless/sessions",
+            method=REQUEST_METHOD_POST,
+            params=session_options,
+            token=workos.api_key,
+        )
+
+    def send_session(self, session_id):
+        """Send a Passwordless Session via email.
+
+        Note, either 'directory' or 'user' must be provided.
```
Comment (rjvani) [inline workos/passwordless.py:46]: Can you remove this?

File: workos/passwordless.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+        Args:
+            session_id (str): The unique identifier of the Passwordless
+      				Session to send an email for.
+
+        Returns:
+            dict: {"success": true} if successful
```
Comment (rjvani) [inline workos/passwordless.py:53]: In our Audit Trail module, we just return `True` directly. I think we should be consistent. @henrylamchan thoughts?

Comment (henrylamchan) [reply workos/passwordless.py:53]: Im down for just returning `True`. We're pretty inconsistent now with the python SDK but we want to move in a direction where we're not passing back a raw response.

Comment (rjvani) [reply workos/passwordless.py:53]: Yeah I agree with that. Let's return `True` here as well.

File: workos/passwordless.py (hunk 1 above)
Comment (zlwaterfield) [reply workos/passwordless.py:46]: - [ ] remove unnecessary comment

File: workos/passwordless.py (hunk 2 above)
Comment (zlwaterfield) [reply workos/passwordless.py:53]: Ok cool, I can do that.

- [ ] return true instead of the `dict`

File: workos/passwordless.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+        pass
+
+    @property
+    def request_helper(self):
+        if not getattr(self, "_request_helper", None):
+            self._request_helper = RequestHelper()
+        return self._request_helper
+
+    def create_session(self, session_options):
+        """Create an Passwordless Session.
```
Comment (rjvani) [inline workos/passwordless.py:20]: s/an/a

**Review by rjvani — APPROVED** [review]

**Pull request merged**